BUILDING.md

```diff
@@ -39,6 +39,7 @@ As the NUnit solution targets multiple frameworks, a single build will generate
  (directory with nunit.sln)
     bin\
        Debug\
+	      net8.0
```

Comment:
Clearly we didn't update this when dropping net5.0, netcoreapp3.1 and netstandard2.0 builds.
Please delete those.

Also check your indentation

Reply:
thank you @manfred-brands  for pointing out this - i have removed the .net5.0, net.3.1 and netstandard2.0